src/libraries/System.Runtime/tests/System/DateTimeOffsetTests.cs

```diff
@@ -285,6 +298,26 @@ public static void Ctor_Int_Int_Int_Int_Int_Int_TimeSpan_Invalid()
             AssertExtensions.Throws<ArgumentOutOfRangeException>(null, () => new DateTimeOffset(max.Year, max.Month, max.Day, max.Hour, max.Minute, max.Second + 1, TimeSpan.Zero));
         }
 
+        [Theory]
+        [InlineData(2000, 1, 1, 12, 34, 59)]
```

Comment:
>InlineData(2000, 1, 1, 12, 34, 59)] [](http://example.com/codeflow?start=9&length=35)

try to add a case like `[InlineData(2022, 12, 31, 23, 59, 59)]` and ensure using non zero offset when creating the DateTimeOffset.